services.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"
+        content="width=device-width, initial-scale=1"
+        name="viewport">
+  <title>Great Idea!</title>
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+        rel="stylesheet">
+  <link href="css/index.css"
+        rel="stylesheet"><!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+</head>
+<body>
+  <div class="container">
+    <header>
+      <nav>
+        <a href="services.html">Services</a> <a href="#">Product</a> <a href="#">Vision</a> <a href="#">Features</a> <a href="about.html">About</a> <a href="#">Contact</a>
+      </nav><a href="index.html"><img alt="Great Idea! Company logo."
+           class="logo"
+           src="img/logo.png"></a>
+    </header><img alt="Our services header image"
+         class="services-header-img"
+         src="img/services-header.jpg">
+    <article class='services'>
+      <h2>Services</h2>
+      <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu
+      consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+      <section class='service-descriptions'>
+        <div class='service-box'>
+          <h3>Digital Design</h3>
+          <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus
+          nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p><a href='#'>Learn More</a>
```

Comment:
The same idea could be applied here! This is better for readability:
```
<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus 
nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
<a href='#'>Learn More</a>
```
or even:
```
<p>
    Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus 
nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
</p>
<a href='#'>Learn More</a>
```